### DISCUSSION TITLE
Home appliances disconnected

### DISCUSSION THREAD

**Discussion by Drahos-X** [alexkn/node-red-contrib-homeconnect#37] [Q&A]
Since yesterday, all my home appliances are disconnected, i.e. "connected: false". Get_home_appliances is working, but nothing else. However, I can see them connected via native Home Connect app - everything ok there.
I really don't know what has happened. Any clue?

**Comment by xerion3800** [comment]
In the Homme connect app I got an info that they have problems with some Server.
